tests/test_serialize.py

```diff
@@ -896,7 +896,7 @@ class UserLogin(User):
         password: SecretStr
 
     class OuterModel(BaseModel):
-        maybe_as_any: Optional[SerializeAsAny[User]]
+        maybe_as_any: Optional[SerializeAsAny[User]] = None
```

Comment:
Not strictly necessary for this PR but the insert_assert() below doesn't work without it